tools/interpolation/run_idw_interpolation.xml

```diff
@@ -0,0 +1,135 @@
+        <requirement type="package" version="1.1.3">r-rcolorbrewer</requirement>
+        <requirement type="package" version="3.6.23">r-raster</requirement>
+        <requirement type="package" version="2.1.1">r-gstat</requirement>
+    </requirements>
+    <command detect_errors='exit_code'><![CDATA[
+    Rscript '$__tool_directory__/run_idw_interpolation.R'
+        --observationsCsv '$observations_csv'
+        --latitudeColumn '$latitude_column'
+        --longitudeColumn '$longitude_column'
+        --observationsColumn '$observations_column' 
+        --studyArea '$study_area'
+        --idwPower '$idw_power' 
+        --samplePoints '$sample_points' 
+        --sampleType '$sample_type'
+        --legendLabel '$legend_label'
+        --legendPosition '$legend_position'
+        --numberClasses '$number_classes'
+        --dotSize '$dot_size'
+        --colorType '$color_type'
+        --testCase '$test_case'  
+        --outputData '$output'
+    ]]></command>
+    <inputs>
+        <param type="data" format="csv" name="observations_csv" label="Select the .csv file including the observations."/>
+        <param type="data" format="geojson" name="study_area" label="Select the .geojson containing the study area."/>
+        <param name="latitude_column" type="data_column" data_ref="observations_csv" label="In which column are the latitude coordinates?"/>
+        <param name="longitude_column" type="data_column" data_ref="observations_csv" label="In which column are the longitude coordinates?"/>
+        <param name="observations_column" type="data_column" data_ref="observations_csv" label="In which column are the observation values?"/>
+        <param value="2" min="0.1" max="5" type="float" name="idw_power" label="Select the interpolation power of the IDW method."/>
+        <param value="50000" min="1000" max="100000" type="integer" name="sample_points" label="Select the number of sample points."/>
+        <param type="select" name="sample_type" label="Select a sample type.">
+            <option value="regular">Regular</option>
+            <option value="random">Random</option>
+            <option value="stratified">Stratified</option>
+            <option value="nonaligned">Nonaligned</option>
+        </param>
+        <param type="text" name="legend_label" label="Enter a label for the legend" optional="false"/>
+        <param type="select" name="legend_position" label="Should the legend be displayed inside or outside the frame?">
+            <option value="false">Inside</option>
+            <option value="true">Outside</option>
+        </param>
+        <param value="0.2" min="0.1" max="2" type="float" name="dot_size" label="Select the dot size of the points shown on the map."/>
+        <param value="10" min="1" max="100" type="integer" name="number_classes" label="Number of classes (Note: Not all values are possible. tmap tries to generate a number of classes which are as close to the indicated value as possible)."/>
```

Comment:
Please keep the label short and include a help="" with more information, like your Note here